packages/eslint-plugin-pf-codemods/lib/rules/v5/emptyState-rename-components.js

```diff
@@ -0,0 +1,10 @@
+const { renameComponents } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8737
+module.exports = {
+  meta: { fixable: "code" },
+  create: renameComponents({
+    "EmptyStatePrimary": "EmptyStateActions",
+    "EmptyStateSecondaryActions": "EmptyStateActions",
+  }),
+};
```

Comment:
Since EmptyStatePrimary and EmptyStateSecondaryActions are being replaced completely, we should try to also remove their imports as part of the fix.

Reply:
@thatblindgeye @gitdallas 

So I have done this fix and I found out there are some issues related to how the `renameComponents` helper works. I also have some questions about alias imports.

1) Is there ever a case where the components are not replaced completely? Maybe let's say that some component becomes deprecated and it gets a new name. What the fixer inside `renameComponents` does, is that it **adds** new import (does not replace the old one), but on this contrary it renames all JSX elements. 

(now I see there is the `data-codemods` attribute which if present, don't rename the component, so I guess it makes sense not to replace the import)

2) If the component is aliased (e.g. `EmptyStatePrimary as ESP`), `renameComponents` helper will not rename the component. This would be fine, but only in the case, if the import would be **replaced** correctly -- so the import would look like `EmptyStateActions as ESP`. Instead it only adds `EmptyStateActions` import (not aliased), does not rename the JSXElements and keeps the import of the old component - so consumer would have to solve this fix manually.

3) In my specific case, when we are renaming both `EmptyStatePrimary` and `EmptyStateSecondaryActions` to the same component `EmptyStateActions`, if both components have aliased imports, let's say:
```
EmptyStatePrimary as Primary,
EmptyStateSecondaryActions as Secondary,
```
can we, after renaming, keep the EmptyStateActions import twice like this?
```
EmptyStateActions as Primary,
EmptyStateActions as Secondary,
```
Compilation seems to works fine.

4) When I try using alias import, I am getting this ReferenceError, do you know what is wrong?

![Screenshot 2023-03-29 at 13 29 27](https://user-images.githubusercontent.com/84135613/228524664-aa3f1577-c78b-407b-9f95-43325a72703c.png)

What do you think? In my opinion the `renameComponents` should be improved. Right now when I am using it, I have to handle removing imports separately and cannot handle the aliased imports. So I will end up not using it. Maybe I am missing something with the `data-codemods` attribute.

Well it was a long comment, I hope it all makes sense :D